### PR TITLE
Added threadcount to unique stack output.

### PR DESCRIPTION
**Pull request by bveldhoen**
Great and useful script!
When there are many threads, it's annoying to have to manually count the number of threads :).

### CODE REVIEW

File: uniq-stacks.py [hunk 1]
```diff
@@ -91,11 +91,11 @@ def invoke(self, argument, from_tty):
         sorter = lambda d: sorted(d.items(), key=lambda item: len(item[1]),
                 reverse=True)
 
-        gdb.write("\n== Printing {} unique stacks from {} threads\n\n".format(
+        gdb.write("\n== Printing {} unique stack(s) from {} thread(s)\n\n".format(
             len(uniq), len(traces)))
 
         for k, v in sorter(uniq):
-            gdb.write("Stack for thread ids {}\n".format(sorted(v)))
+            gdb.write("Unique stack common for {} thread(s); Thread id(s): {}\n".format(len(v), sorted(v)))
```
Comment (flandr) [inline uniq-stacks.py]: Nit: this phrasing is a bit redundant, IMO. Would you mind removing the leading "Unique"?

**Review by flandr — COMMENTED** [review]
Thanks, and LGTM but please adjust phrasing as suggested.

**Comment by bveldhoen** [comment]
I had to google "LGTM" :).
Fixed!

**Review by flandr — APPROVED** [review]

**Comment by flandr** [comment]
Thanks!

**Pull request merged**